Blake fusick hw2

Fusickb opened the pull request.

Fusickb committed:
Lots of Changes

Updating assignment two for the code that I had

Fusickb committed:
Opening File Error Handling and Slider

Added error handling so it wouldn't crash if encountering an error when opening a file. Also added a slider under the graph to change the number of histogram bins.

jeremydaily reviewed (changes requested):
Remove all globals.
The areas under the other curves don't look right. You'll have to estimate the lognormal on only positive values.

Fusickb committed:
Removed global variable dependence

Removed all global variables and replaced them with private variables and instance variables.

Fusickb committed:
Cleaned up code

Deleted a lot of useless lines and added comments

Fusickb committed:
Fixed lognorm added some other distribution code

Finished lognorm, and started adding other distributions.
